CMSgov/design-system
# [WNMGDS-3135] Calculates unit test code coverage

## Summary
- [WNMGDS-3135](https://jira.cms.gov/browse/WNMGDS-3135)
- This combines the coverage output when running react and web component unit tests and generates three json files in the `tests/unit/coverage-data` directory when running `yarn test:unit:coverage:all`:

  1. `coverage-summary-react.json`
  2. `coverage-summary-wc.json`
  3. `coverage-summary.json`
- Additional scripts have been added to the package.json in case there's a need for determining discrete code coverage
## How to test

1. Optionally, pull this branch down and verify that `yarn  test:unit` and `yarn test:unit:wc` all work normally
2. Pull this branch down and run `yarn test:unit:coverage:all`
3. Try out `yarn test:unit:coverage:wc`; verify that a new json file called `coverage-summary-wc.json` is created in the `tests/unit/coverage-data` directory
4. Try out `yarn test:unit:coverage:react`; verify that a new json file called `coverage-summary-react.json` is created in the `tests/unit/coverage-data` directory

## Checklist



### If this is a change to code:

- [x] Verified that running both `yarn test:unit` and `yarn test:browser:all` were each successful
